### DISCUSSION TITLE
configure HTTPS

### DISCUSSION THREAD

**Discussion by smpreston162** [Corsinvest/cv4pve-admin#76] [Q&A] [answered]
what is the recommend method for this application to deploy with HTTPS?

**Comment by franklupo** [comment, marked as answer]
Hi,
at this moment it is better to use a container with Nginx acting as a reverse proxy. Install certificates or use Let's Encrypt.

An example without SSL reverse proxy on port 4000 https://github.com/Corsinvest/cv4pve-admin/tree/main/src/Docker/Nginx/ReverseProxy
